docs: new tutorial and updates related to RoundRequestor. (#54)
algorand/conduit: docs/tutorials/IndexerMigration.md

    "netaddr": $ALGOD_ADDR,
    "token": $ALGOD_TOKEN,
    "mode": "follower",
    catchup-config:
        catchpoint: ""
        admin-token: ""
exporter:
  name: "postgresql",
  config:
    "connection-string": $PGSQL_CONNECTION_STRING,
```

> If your algod node needs to run fast catchup, you can fill in the catchup-config sections. You'll need to first lookup your Indexer round from the postgres database. The Indexer stores the latest round in the database, and you can read it via the `/health` endpoint. The result is formatted in json

Reviewer: ```suggestion
If your algod node needs to run fast catchup, you can fill in the catchup-config section. You'll need to first look up your Indexer round from the postgres database. The Indexer stores the latest round in the database, and you can read it via the `/health` endpoint. The result is formatted in json
```

so you can use jq to more easily see your Indexer's round (if your Indexer is listening locally on port 8980).
```bash
curl http://localhost:8980/health | jq '.round'
```

> Now you can look up a catchpoint, conduit will run fast catchup on your node if a catchpoint is provided. Lookup the closest catchpoint prior to the desired sync round. 

Reviewer: ```suggestion
Now that you can look up a catchpoint, conduit will run fast catchup on your node if a catchpoint is provided. Look up the closest catchpoint prior to the desired sync round. 
```

For a list of catchpoints, you can reference the following:
* [Mainnet](https://algorand-catchpoints.s3.us-east-2.amazonaws.com/consolidated/mainnet_catchpoints.txt)
* [Testnet](https://algorand-catchpoints.s3.us-east-2.amazonaws.com/consolidated/testnet_catchpoints.txt)
* [Betanet](https://algorand-catchpoints.s3.us-east-2.amazonaws.com/consolidated/betanet_catchpoints.txt)

For example, if your postgres database is on round 25001234, use the following configuration:
```yaml
    catchup-config:
        catchpoint: "25000000#EOX5UYQV4IXTGYQCIDR7ZLUK6WZGDC5EG6PYQGBG6PBYNAQUPN2Q"
        admin-token: "$ALGOD_ADMIN_TOKEN"
```

Then run Conduit, `conduit -d $CONDUIT_DATA_DIR`!

You can separately run your Indexer with `--no-algod` to connect your API to the database.

> If you configured a catchpoint, Conduit will facilitate that process during initialization. Once the catchpoint has been reached the node will resume normal catchup until it reaches the target round defined in postgres. If your catchpoint is not exactly the round your database is on,

Reviewer: ```suggestion
If you configured a catchpoint, Conduit will facilitate the fast catchup process during initialization. Once the catchpoint has been reached the node will resume normal catchup until it reaches the target round defined in postgres. If your catchpoint is not exactly the round your database is on,
```

> you will need to wait for normal catchup to get your node to the desired point in time.

Reviewer: this last sentence seems redundant with the previous one?
